src/GlobalAssemblyInfo.cs

```diff
@@ -13,7 +13,7 @@
 [assembly: AssemblyProduct("R Tools for Visual Studio")]
 [assembly: AssemblyDescription("")]
 
-#if !NETSTANDARD1_6
+#if !NETSTANDARD1_6 && !NETCOREAPP1_1
```

Comment:
For the Linux version of the containers.

---
In reply to: [129090290](https://github.com/Microsoft/RTVS/pull/3734#discussion_r129090290) [](ancestors = 129090290)